Domain/Aggreagtes/StudentAggregate/Student.cs

```diff
@@ -16,18 +18,20 @@ public class Student : AuditableEntity, IAggregateRoot
         public DateOnly? DateOfBirth { get; private set; }
         public Address? Address { get; private set; }
         public EducationLevel? Education { get; private set; }
-
         public Guid? CourseId { get; private set; }
```

Comment:
By the requirements, a students can take more than a course with us. I expect a List of courses or better List of enrollment. There could be a business logic to restrict the student to one active course at a time.

Reply:
i will work on the enrollment class and modified the student class